js/admin.js

```diff
@@ -0,0 +1,13 @@
+jQuery( document ).ready( function ( $ ) {
```

Comment:
Yeah thought about that. But jQuery loads in the admin by default anyway. I don't think that's going away any time soon.

I can rewrite it to vanilla js